kit/src/components/indicator/style.scss

```diff
@@ -20,3 +20,16 @@
 .indicator-idle {
 	stroke: var(--warning-light);
 }
+
+.online-circle {
```

Comment:
Here, we can do `.indicator-idle.filled { color: whatever }` and `.indicator-online.filled { color: whatever }`